src/System.IO.Compression/tests/ZipArchive/zip_CreateTests.cs

```diff
@@ -115,6 +115,48 @@ public static async Task CreateNormal_Unicode_Unseekable()
                 IsZipSameAsDir(s, zfolder("unicode"), ZipArchiveMode.Read, requireExplicit: true, checkTimes: true);
             }
         }
+
+        [Fact]
+        public static void CreateUncompressedArchive()
+        {
+            using (var s = new MemoryStream())
+            {
+                var testfilename = "testfile";
+                var testFileContent = "Lorem ipsum dolor sit amet, consectetur adipiscing elit.";
+                using (var zip = new ZipArchive(s, ZipArchiveMode.Create))
+                {
+                    var utf8WithoutBom = new Text.UTF8Encoding(false);
+                    var newEntry = zip.CreateEntry(testfilename, CompressionLevel.NoCompression);
+                    using (var writer = new StreamWriter(newEntry.Open(), utf8WithoutBom))
+                    {
+                        writer.Write(testFileContent);
+                        writer.Flush();
+                    }
+                    var fileContent = s.ToArray();
+                    // zip file header stores values as little-endian
+                    var compressionMethod = fileContent[8];
+                    Assert.Equal(0, compressionMethod); // stored => 0, deflate => 8
```

Comment:
This assert is failing:
https://mc.dot.net/#/user/Marv51/pr~2Fjenkins~2Fdotnet~2Fcorefx~2Fmaster~2F/test~2Ffunctional~2Fcli~2F/1ccf010a41e3ad7031e1339e309cd2d95f830c3e/workItem/System.IO.Compression.Tests/analysis/xunit/System.IO.Compression.Tests.zip_CreateTests~2FCreateUncompressedArchive

```text
Message :
Assert.Equal() Failure
Expected: 0
Actual:   8
Stack Trace :
   at System.IO.Compression.Tests.zip_CreateTests.CreateUncompressedArchive() in D:\j\workspace\windows-TGrou---2a8f9c29\src\System.IO.Compression\tests\ZipArchive\zip_CreateTests.cs:line 139
```